api/src/utils/helpers/URL.ts

```diff
@@ -1,6 +1,10 @@
 export const buildURL = (baseUrl: string, relativePath: string): string => {
   try {
-    const url = new URL(relativePath, baseUrl);
+    const normalizedBaseUrl = baseUrl.endsWith('/') ? baseUrl : `${baseUrl}/`;
```

Comment:
Hi @devamitranjan, i can notice that this logic will expect to have the relativePath prop to be relative, what if we receive an absolute one.
`buildURL('http://my-domain.com/api/','http://github.com/'); `
The output will be => http://github.com/

Reply:
Okay @yassinedorbozgithub added the change.